jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/wizard/SamProjectWizard.kt

```diff
@@ -138,12 +137,10 @@ abstract class SamProjectTemplate {
         }
     }
 
-    fun getIcon() = AwsIcons.Resources.SERVERLESS_APP
```

Comment:
what does removing this do?

Reply:
We put an icon next to each template..it felt odd...

Reply:
https://github.com/aws/aws-toolkit-jetbrains/pull/2112/files#diff-3cdf10103b9290aec22fe7703ab30973